src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,71 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String fruitStatistic = readFile(fromFileName);
+        String report = createReport(fruitStatistic);
+        writeFile(report, toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            int value = reader.read();
+            while (value != -1) {
+                builder.append((char) value);
+                value = reader.read();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + fromFileName, e);
+        }
+        return builder.toString();
+    }
+
+    private String createReport(String fruitStatistic) {
+        final String delimiter = System.lineSeparator();
+        final String recordSeparator = ",";
+        final String supplyAction = "supply";
+        final String buyAction = "buy";
+        final String resultAction = "result";
```

Comment:
all these parameters should be constants
```suggestion
        final String recordSeparator = ",";
        final String supplyAction = "supply";
        final String buyAction = "buy";
        final String resultAction = "result";
```